src/risclog/sqlalchemy/tests/test_db.py

```diff
@@ -119,26 +119,35 @@ class Model_1(Base_1):
 
     class ObjectBase_2(ObjectBase):
         _engine_name = 'db2'
-    declarative_base(ObjectBase_2)
+    Base_2 = declarative_base(ObjectBase_2)
+
+    class Model_2(Base_2):
+        id = Column(Integer, primary_key=True)
 
     def tearDown():
         risclog.sqlalchemy.db.unregister_class(Model_1)
     request.addfinalizer(tearDown)
 
     database_1.create_all('db1')
+    database_2.create_all('db2')
 
     db = get_database(testing=True)
     # It used to be necessary to bind the session to the database to be
     # queried if more than one database was being used. This is no longer the
     # case as of SQLAlchemy 1.0.
     assert db.session.query(Model_1).count() == 0
 
+    Model_1.create().persist()
+    Model_2.create().persist()
+    import transaction
+    transaction.commit()
+
     # When using session.execute, a manual bind is necessary though
     with pytest.raises(RuntimeError):
         assert db.session.execute('SELECT count(*) FROM model_1').fetchall()
     # Using a bound session leads to a result:
     assert db.session.using_bind('db1').execute(
-        'SELECT count(*) FROM model_1').fetchall() == [(0,)]
+        'SELECT count(*) FROM model_1').fetchall() == [(1,)]
```

Comment:
This kind of changes the test outcome, but without any change in the name or docstring, what the test is about. I am also not sure, what would be different if `Model_2` would not be persisted.

Reply:
Sorry, I can't remember all the details of the changes either, but looking at it now it seems to me that the test before my changes didn't test what it says. It says `# Using a bound session leads to a result:` and then at the end it checks that `count(*)` == 0. Now there is really a result.